siliconcompiler/core.py

```diff
@@ -2540,18 +2522,17 @@ def check_logfile(self, step=None, jobname=None, index=None,
         '''
 
         # Using manifest to get defaults
-        if step is None:
-            step = self.get('arg', 'step')
-        if index is None:
-            index = self.getkeys('flowgraph', step)[0]
```

Comment:
Any reason for getting rid of this?

Reply:
Adding it back, should be safe (with the flow added)